src/identity/Identity.ts

```diff
@@ -100,20 +192,101 @@ export default class Identity extends PublicIdentity {
 
   private readonly signKeyringPair: KeyringPair
   private readonly boxKeyPair: BoxKeyPair
+  /**
+   * @description Creates a new instance of public identity.
+   * @returns An `address` which identity's a specific account on the chain.
```

Comment:
Not sure about the grammar 🙃

Reply:
Yes, you are right 👍 . I changed it and hopefully a little more clear.